### DISCUSSION TITLE
Paramount +

### DISCUSSION THREAD

**Discussion by MrLucado** [Dreamlinerm/Netflix-Prime-Auto-Skip#64] [General]
I just wanted to share that this works flawlessly on Paramount + as well. I noticed it wasn't mentioned anywhere and also isn't included in the built-in options. But it does it's job, no issues.

**Comment by Dreamlinerm** [comment]
what do you mean in it works on paramount+? I am pretty sure it doesnt unless its integrated into some other site.

**Comment by Dreamlinerm** [comment]
ah you watch paramount+ over amazon yeah then it works. It works on all amazon chanells

**Comment by Dreamlinerm** [comment]
> I just wanted to share that this works flawlessly on Paramount + as well. I noticed it wasn't mentioned anywhere and also isn't included in the built-in options. But it does it's job, no issues.

or do you mean on their individual page?

**Comment by Dreamlinerm** [comment]
I can add it to the discription but I think I consider it under prime video? or do you think i should add it specifically?
